remote-expectations.md

```diff
@@ -1,23 +1,20 @@
-_Jumpstart Live (JSL)_
+_JumpStart Live (JSL)_
 # Remote-Student Expectations
 
 ## Overview
-As a remote-student participating in Jumpstart Live you will be assigned a group with between 2 – 3 other students. You will also be assigned a mentor from the previous Ada cohort. You will work through all of the Jumpstart Live curriculum individually and then meet with your group over video chat to discuss your answers, answer questions, and support each other through the learning process.
-
-## Participating Students
-We expect students participating in Jumpstart live to not yet be comfortable using loops and arrays. We ask our more experienced Ada students to leave this space for those still learning how to use loops and arrays effectively, so they can feel confident in the material that you already know before the first day of Ada classes.
+As a remote-student participating in JumpStart Live you will be assigned a group with between 2 – 3 other students. You will also be assigned a mentor from the previous Ada cohort. You will work through all of the JumpStart Live curriculum individually and then meet with your group over video chat to discuss your answers, answer questions, and support each other through the learning process.
 
 ## Communication
-You are expected to check your groups Slack channel daily and be responsive and receptive to your group members needs and questions. You should be communicating with your mentor at least two times a week in similar fashion.
+You are expected to check your groups Slack channel daily and be responsive and receptive to your group members' needs and questions. You should be communicating with your mentor at least two times a week in similar fashion.
 
 ## Video Chats
 As a group you should be having video chats for at least 2 hours each week. It is up to your group to determine if there will be multiple short sessions each week, or one long session each week. Your mentor should be attending and participating in at least one video chat a week.
 
 ## Work Submissions
-When you have completed all the work for a particular day, you should send a gist to the lead Jumpstart Instructor on slack. There are 6 total days, so this should be 6 total gists, but feel free to split the work up into more gists if that makes sense for you.
+When you have completed all the work for a particular day, you should send a gist to the lead JumpStart Instructor. There are 6 total days, so this should be 6 total gists, but feel free to split the work up into more gists if that makes sense for you.
```

Comment:
Change to _at least 6 gists_ if you are allowing them to do more

Reply:
Done.